testing/unit-testing-jest/docs/getting-dom-elements.md

```diff
@@ -0,0 +1,63 @@
+
+# How to get elements from the DOM on testing
+Check if an element render the correct information on the page is very important to test and more if we are frontend developers, so to test the elements you need to access to the elements and also call his events.
```

Comment:
It is a kind of heavy to read. You use too many times "elements". Check your punctuation 😄